docs/usage/configuration/config_documentation.md

```diff
@@ -2873,6 +2873,9 @@ Use this setting to enable password-based logins.
 
 This setting has the following sub-options:
 * `enabled`: Defaults to true.
+   Set to false to disable password authentication.
+   Set to `only_for_reauth` to allow users with existing passwords to use them
+   to log in and reauthenticate, whilst preventing new users from setting passwords.
```

Comment:
Do we also want to update the documentation in the sample config?

Reply:
Ah, I thought it got removed but no you're right